routes/api.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET api page. */
+router.get('/', function(req, res, next) {
+  res.json({success : true});
+});
+
+module.exports = router;
```

Comment:
Notice how Github is showing no newline at the end of a file. This is a way to make sure a file wasn't cut off randomly somewhere so it's good practice to have every file end with a newline.